.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,23 @@
 <!-- Provide a general summary of your changes in the Title above -->
 
+
 ## Description
-<!-- Describe your changes in detail -->
+<!-- Describe your changes in detail. -->
+<!-- Please indicate if this change breaks or may break existing functionality, so we can mark it as 'breaking'. -->
```

Comment:
I think, especially for new or inexperienced contributors, this is very hard to assess.

```suggestion
<!-- If possible, please indicate if this change breaks or may break existing functionality, so we can mark it as 'breaking'. -->
```